GMGoogleDrive/Public/Add-GDriveItem.ps1

```diff
@@ -85,6 +85,18 @@ function Add-GDriveItem {
         [Alias('Metadata')]
         [string]$JsonProperty = '',
 
+        [ValidateSet("*",'kind','id','name','mimeType',
+        'description','starred','trashed','explicitlyTrashed','parents','properties','appProperties','spaces','version',
+        'webContentLink','webViewLink','iconLink','thumbnailLink','viewedByMe','viewedByMeTime','createdTime','modifiedTime',
+        'modifiedByMeTime','sharedWithMeTime','sharingUser','owners','lastModifyingUser','shared','ownedByMe',
+        'viewersCanCopyContent','writersCanShare','permissions','originalFilename','fullFileExtension',
+        'fileExtension','md5Checksum','sha1Checksum','sha256Checksum','size','quotaBytesUsed','headRevisionId','contentHints',
+        'imageMediaMetadata','videoMediaMetadata','capabilities','isAppAuthorized','hasThumbnail','thumbnailVersion',
+        'modifiedByMe','trashingUser','trashedTime','teamDriveId','hasAugmentedPermissions',
+        'keepForever', 'published', # revisions
+        IgnoreCase = $false)]
+        [string[]]$ResultProperty = @('kind','id','name','mimeType','parents'),
```

Comment:
Copy-GDriveItem
Find-GDriveItem
Move-GDriveItem
New-GDriveItem
New-GDriveShortcut

These cmdlets also need this?
Find already have -Property, i think -ResultProperty may be added as alias

Reply:
I just want have a clear difference between $JsonProperty and $Property. This is why I used ResultProperty. It's fine for me to use Property

Reply:
ResultProperty changed to Property in fa952e4